### ISSUE TITLE
udacity course

### ISSUE THREAD

**Issue by huhuchen** [huhuchen/Learning-ML#1]

**Comment by huhuchen** [comment]
朴素贝叶斯练习
梯度下降 gradient descent
线性回归  linear regression
最小二乘法  least squares
逻辑回归 logistic regression
支持向量机 support vector machine
神经网络 neural network
